dimagi/pages/static/style/pages/blog.scss

```diff
@@ -63,12 +59,6 @@ a.post {
 
 .powerful-data {
   $_base-height: 175px;
-  $_height-by-size: (
-    extra-small: 205px,
-    small: 270px,
-    medium: 210px,
-    large: 265px,
-    extra-large: 320px,
-  );
+  $_height-by-size: ( extra-small: 205px, small: 270px, medium: 210px, large: 265px, extra-large: 320px, );
```

Comment:
just fYI. the best practices for our `scss` code, and this is even present in the `scss` lint file that you should link with your editor, is that EACH property must be on a separate line. Please change this back to what it was. Thanks